Source/SourceKittenFramework/StringView.swift

```diff
@@ -125,7 +125,9 @@ public struct StringView {
      Returns a substring from a start and end SourceLocation.
      */
     public func substringWithSourceRange(start: SourceLocation, end: SourceLocation) -> String? {
-        return substringWithByteRange(start: Int(start.offset), length: Int(end.offset - start.offset))
+        let byteRange = ByteRange(location: ByteCount(Int(start.offset)),
+                                  length: ByteCount(Int(end.offset - start.offset)))
```

Comment:
here as well, we can use `ByteCount(Int(start.offset)) -> ByteCount(start.offset)`

Reply:
Int32.